src/image_surface.rs

```diff
@@ -44,6 +110,31 @@ impl ImageSurface {
         }
     }
 
+    #[cfg(feature = "png-functions")]
+    pub fn create_from_png_stream<R: Read>(stream: R) -> io::Result<ImageSurface> {
```

Comment:
Hm, I'm not sure I follow... I can just omit the `Path` version? Or, if not, what should it be named?